README.md

```diff
@@ -42,6 +42,18 @@ If you want to use [PROJ](https://proj.org/) in your code, you can if you
 activate the `proj` feature (`cargo build --features=proj`). Then don't forget
 to protect your code with `#[cfg(feature="proj")]`.
 
+### Feature `xmllint`
+`transit_model` is capable of exporting NeTEx France format. In the tests, we're
+automatically verifying that the produced files are matching the NeTEx
+specification.  For that, we're using the tool `xmllint` which can be install
+on Debian with the package `libxml2-utils`. Therefore, these tests are run only
+if you activate them.
+
+```bash
+apt install libxml2-utils
+cargo test --features xmllint
```

Comment:
I did not look for the cause, but the command does not pass

Reply:
It should work now. I had a missing dependency on `proj` feature. See 87e7c10bc0c4bfe896dd1fb23708be0ef6ae5c7d.